src/Visitors/AddArrayPropertyItem.php

```diff
@@ -31,7 +36,7 @@ protected function updateNode(Node $node): void
             throw new UnexpectedPropertyTypeException(
                 property: $this->name,
                 expectedType: 'array',
-                actualType: (is_null($node->type)) ? 'null' : (string) $node->type,
+                actualType: $node->type !== null ? (string) $node->type : 'null',
```

Comment:
it's not good to use a negative condition (type not null), so please revert this change